test/controllers/links_controller_test.rb

```diff
@@ -26,6 +26,37 @@ class LinksControllerTest < ActionController::TestCase
       assert_redirected_to /my_plugin/
     end
 
+    test '#documentation_url receives an allowed root_url domain' do
+      ['theforeman.org', 'redhat.com', 'orcharhino.com'].each do |domain|
```

Comment:
this could use `LinksController::TRUSTED_DOMAINS`, but I am not sure I like that, as that would mean that if that list ever becomes empty, the test will still pass?